services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/extensions/dynamodb/mappingclient/functionaltests/AsyncBasicCrudTest.java

```diff
@@ -228,6 +226,25 @@ public void deleteTable() {
                                 .join();
     }
 
+    @Test
+    public void getItemInterfaceToOperationSucceeds() {
+        mappedTable.getItem(r -> r.key(Key.create(stringValue("id-value"), stringValue("sort-value")))).join();
+        mappedTable.getItem(GetItemEnhancedRequest.builder()
+                                                  .key(Key.create(stringValue("id-value"), stringValue("sort-value")))
+                                                  .build()).join();
+    }
+
+    @Test
+    public void putItemInterfaceToOperationSucceeds() {
+        Record record = new Record()
+            .setId("id-value")
+            .setSort("sort-value")
+            .setAttribute("one");
+
+        mappedTable.putItem(Record.class, r -> r.item(record)).join();
+        mappedTable.putItem(PutItemEnhancedRequest.builder(Record.class).item(record).build()).join();
+    }
```

Comment:
I think we should be able to make basic assertions here beyond 'it didn't blow up'.

Reply:
I could not put my heart into the wiring tests after all and I removed them. All wiring combinations are still tested (no unused interface methods). We can always rethink later.